Makefile

```diff
@@ -27,16 +27,31 @@ endif
 
 include sys/make/make.def
 
-##################################################
+###################################################
 # It is possible to annotate a set of results with
 # a LOG_NOTE which will be added to the log header
 # files
-#################################################
+###################################################
 LOG_NOTE?="none"
 
-##################################################
+###################################################
+# Thread & Team Numbers
+###################################################
+ifdef OMPVV_NUM_THREADS_DEVICE
+	DTHREADS = -DOMPVV_NUM_THREADS_DEVICE=${OMPVV_NUM_THREADS_DEVICE}
+endif
+
+ifdef OMPVV_NUM_TEAMS_DEVICE
+	DTEAMS = -DOMPVV_NUM_TEAMS_DEVICE=${OMPVV_NUM_TEAMS_DEVICE}
+endif
+
+ifdef OMPVV_NUM_THREADS_HOST
+	HTHREADS = -DOMPVV_NUM_THREADS_HOST=${OMPVV_NUM_THREADS_HOST}
+endif
+
+###################################################
```

Comment:
Shall we have these in /sys/make.def or here?

Reply:
Decided to move them to make.def